bitnami/wordpress/README.md

```diff
@@ -499,6 +498,10 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Notable changes
 
+### 13.2.0
+
+- Removed support for limiting auto-updates to WordPress core via the `wordpressAutoUpdateLevel` option. To update WordPress core, we recommend to swap the container image version for your deployment instead of using the built-in update functionality.
```

Comment:
More than swapping the container image, for upgrading WordPress the recommendation should be to upgrade the Helm chart itself because it is possible changes in the application involve changes in the Helm chart.

Apart from that, please note we are not using lists for the notable change, the initial `-` can be safely removed

Reply:
Thanks for the suggestion. I just applied the changes.